### PR TITLE
chore(release-health): adjust available platform list

### LINKED ISSUE #76529: "This project doesn't support sessions" error message when creating an alert
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. go to the create alert page (https://slug.sentry.io/alerts/wizard/?referrer=alert_stream)
2. select a unity project 
3. Add the Percent of sessions affected by an issue is ... condition
4. this message appears:

Image

### Expected Result

users should be able to create this type of request since unity is supported according to the docs: https://docs.sentry.io/product/releases/setup/#release-health

### Actual Result

users cannot currently create this type of request

### Product Area

Alerts

### Link

_No response_

### DSN

_No response_

### Version

_No response_

### PR DESCRIPTION
**Pull request by obostjancic**
Aligns release health platform list in product with the Product availability matrix 

- closes https://github.com/getsentry/sentry/issues/76529

### CODE REVIEW

**Review by ArthurKnaus — APPROVED** [review]

**Comment by vgrozdanic** [comment]
There is another occurrence of rust in the list at line 371

File: static/app/data/platformCategories.tsx [hunk 1]
```diff
@@ -378,20 +374,6 @@ export const releaseHealth: PlatformKey[] = [
...
-  'electron',
-  'javascript-electron',
-  'rust',
-  'php',
-  'php-laravel',
-  'php-symfony',
-  'dotnet',
-  'dotnet-awslambda',
-  'dotnet-gcpfunctions',
-  'dotnet-maui',
```
Comment (vgrozdanic) [inline static/app/data/platformCategories.tsx:390]: Why is this deleted, according to the excel sheet, this is automatically supported?

File: static/app/data/platformCategories.tsx [hunk 2]
```diff
@@ -378,20 +374,6 @@ export const releaseHealth: PlatformKey[] = [
...
-  'dotnet-uwp',
-  'dotnet-wpf',
-  'dotnet-winforms',
-  'dotnet-xamarin',
```
Comment (vgrozdanic) [inline static/app/data/platformCategories.tsx:394]: Why is this deleted, according to the excel sheet, this is automatically supported?
